### PR TITLE
feat: improve formatting for generated markdown in readme files

### PR DESCRIPTION
**Pull request by williamw2**
# Pull Request

## 📖 Description
This fixes a number of formatting issues with the markdown generated by the "cem-markdown" script when rendered in the fast.design web site.

- This removes the unnecessary file location header so that the header hierarchy works better in the right-hand navigation tree.
- Replaces JSDOC @links found in comments with markdown links.
- Removes unnecessary character escapes rendered by @custom-elements-manifest/to-markdown that break formatting.

image

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: change/@microsoft-fast-foundation-41b4c73a-f128-4ac6-974d-2e866cbaebd6.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Improved README markdown formatting\"",
```
Comment (janechu) [inline change/@microsoft-fast-foundation-41b4c73a-f128-4ac6-974d-2e866cbaebd6.json:3]:
```suggestion
  "comment": "Improved README markdown formatting",
```

File: packages/web-components/fast-foundation/src/accordion/README.md [hunk 2]
```diff
@@ -111,20 +111,14 @@ export const myAccordionItem = AccordionItem.compose<AccordionItemOptions>({
...
-
 #### Fields
 
 | Name            | Privacy | Type                                  | Default | Description                                                                                                                                                                         | Inherited From    |
 | --------------- | ------- | ------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
 | `expandmode`    | public  | `AccordionExpandMode`                 |         | Controls the expand mode of the Accordion, either allowing single or multiple item expansion.                                                                                       |                   |
-| `$presentation` | public  | `ComponentPresentation \| null`       |         | A property which resolves the ComponentPresentation instance for the current component.                                                                                             | FoundationElement |
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/accordion/README.md:125]: I think using `|` is preferable, we will run into issues if we attempt to make all TS output "human-readable".

**Review by chrisdholt — APPROVED** [review]

File: packages/web-components/fast-foundation/src/accordion/README.md (hunk 2 above)
Comment (williamw2) [reply packages/web-components/fast-foundation/src/accordion/README.md:125]: This change is only made to the markdown generated by the "to-markdown" library which is only meant for humans to read. The custom-elements.json is unchanged.

File: change/@microsoft-fast-foundation-41b4c73a-f128-4ac6-974d-2e866cbaebd6.json (hunk 1 above)
Comment (williamw2) [reply change/@microsoft-fast-foundation-41b4c73a-f128-4ac6-974d-2e866cbaebd6.json:3]: Good catch. Not sure how that happened.

File: packages/web-components/fast-foundation/CEMToMarkdown.mjs [hunk 3]
```diff
@@ -95,13 +95,30 @@ for(var i = 0, modulesLength = modules.length; i < modulesLength; i++)
...
+        markdown = markdown.replaceAll("https\\:", "https:");
+        // Fix escape of . in some urls
+        markdown = markdown.replaceAll("www\\.w3", "www.w3");
+        // Fix escape of open bracket on links
+        markdown = markdown.replaceAll("\\[", "[");
+        // Fix escape of open paren on links
+        markdown = markdown.replaceAll("\\(", "(");
+
+        // Replace \| with 'or'
+        markdown = markdown.replaceAll("\\|","or");
```
Comment (janechu) [inline packages/web-components/fast-foundation/CEMToMarkdown.mjs:121]: I think this is what's doing the replacement from my comment https://github.com/microsoft/fast/pull/5801/files#r843380813

**Review by janechu — APPROVED** [review]

**Pull request merged**